# The port to be used
port: 12321

# Rate of published messages in Hz
rate: 1

# average walking speed in [m/s]
avg_walking_speed: 0.01

# set the color of your team
# 2: red
# 3: blue
# other values are invalid
team_color: 2

#lifetime of data [s]
lifetime: 3

#threshold for necessary belief in data to be send to teammates [0,1]
belief_threshold: 0

# ROS Topics
team_data: 'team_data'
strategy: 'strategy'
robot_state: 'robot_state'
position: 'pose_with_certainty'
ball: 'balls_relative'
obstacles: 'obstacles_relative'
